.appveyor.yml

```diff
@@ -0,0 +1,32 @@
+environment:
+  matrix:
+  - julia_version: 1
+  - julia_version: 1.1
+  - julia_version: 1.2
+  - julia_version: 1.3
+
+platform:
+  #- x86 # 32-bit
```

Comment:
i have suggested to comment 32-bit as i deem unnecessary to test in that platform..